traffic_ops/testing/api/v4/deliveryservices_test.go

```diff
@@ -296,6 +296,7 @@ func DeleteCDNOldSSLKeys(t *testing.T) {
 	if err != nil {
 		t.Fatalf("couldn't snap CDN: %v", err)
 	}
+	time.Sleep(5 * time.Second)
```

Comment:
instead of a simple sleep, can we use a shorter sleep in the retry loop and check each time? When using Postgres, the update could complete much earlier than Riak, so the tests wouldn't have to wait as long

Reply:
Should be fixed now